test/e2e/standard/standard_test.go

```diff
@@ -195,47 +246,52 @@ var _ = Describe("KVRocks standard controller", func() {
 		}, timeout, interval).Should(Succeed())
 	})
 
-	It("test clean up", func() {
-		Expect(suite.K8sClient.Delete(suite.CTX, instance)).Should(Succeed())
-		Eventually(func() bool {
-			return k8serr.IsNotFound(suite.K8sClient.Get(suite.CTX, key, instance))
-		}, timeout, interval).Should(Equal(true))
-	})
 })
 
 func checkKVRocks(instance *kvrocksv1alpha1.KVRocks) error {
 	password := instance.Spec.Password
 	replicas := int(instance.Spec.Replicas)
 	masterIP := []string{}
 	masterOfSlave := map[int]string{}
+
 	for index := 0; index < replicas; index++ {
 		var pod corev1.Pod
 		key := types.NamespacedName{
-			Namespace: instance.Namespace,
-			Name:      fmt.Sprintf("%s-%d", instance.Name, index),
+			Namespace: instance.GetNamespace(),
+			Name:      fmt.Sprintf("%s-%d", instance.GetName(), index),
+		}
+		if err := env.Client.Get(ctx, key, &pod); err != nil {
+			return err
 		}
-		if err := suite.K8sClient.Get(suite.CTX, key, &pod); err != nil {
+		// portforward for e2e test
+		localPort := []string{fmt.Sprintf("%d:%d", 6379+index, 6379)}
```

Comment:
@tianshimoyi Hi, if I want to connect to Kvrocks in e2e tests, are there any other elegant methods besides portforward?

Reply:
I will remove that by using the telepresence tool.